graylog2-inputs/pom.xml

```diff
@@ -61,7 +61,7 @@
 
         <dependency>
             <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka_2.9.2</artifactId>
+            <artifactId>kafka_2.11</artifactId>
```

Comment:
graylog2-inputs and graylog2-rest-client don't have overlapping dependencies, so it's fine.